pkg/controller/imagebuildmessage/component/amqpmessenger.go

```diff
@@ -201,6 +202,7 @@ func (c *AMQPMessengerComponent) Reconcile(ctx *core.Context) (ctrl.Result, erro
 				images = append(images, reference.TagNameOnly(named).String())
 			}
 			message.ImageURLs = images
+			message.Annotations["imagebuilder.dominodatalab.com/compressed-image-size"] = ib.Status.CompressedImageSize
```

Comment:
can we include the units in the name of the annotation and go var name? e.g. `bytes`

Reply:
done